Wrappers/Python/cil/utilities/display.py

```diff
@@ -92,6 +98,288 @@ def save(self,filename, **kwargs):
         except:
             print("Unable to save image")
 
+
+class show1D(show_base):
+    """
+    This creates and displays 1D plots of pixel values by slicing
+    multi-dimensional data.
+
+    The behaviour is as follows: if provided multiple datasets and a single
+    slice set (see first example below), one line plot will be generated
+    per dataset; if provided a single dataset and multiple sets of slices
+    (see second example below), one line plot will be generated per slice
+    set;  if provided multiple datasets and multiple slice sets, the
+    :math:`i`-th set of slices will apply to the :math:`i`-th dataset, with
+    a line plot generated in each case.
+
+    Parameters
+    ----------
+    data : DataContainer, list of DataContainer, tuple of DataContainer
+        Multi-dimensional data to be reduced to 1D.
+    line_coords : list of tuple or list of list of tuple, default=None
+        A list, or nested list, of (dimension, coordinate) pairs for
+        slicing `data` (default is None, which is only valid when 1D
+        data is passed)
+    label : str, list of str, default=None
+        Label(s) to use in the plot's legend
+    title : str, default None
+        A title for the plot
+    color : str, list of str, default=None
+        Color(s) for each line plot
+    ls : {"-","--","-.",":"}, dict of {"-","--","-.",":"}, default=None
```

Comment:
```suggestion
    ls : {"-","--","-.",":"}, list of {"-","--","-.",":"}, default=None
```

From trying it out, I noticed it is expecting a list, not a dict

Reply:
It also seems to work with a tuple, but just saying list is fine I think

Reply:
I did in fact mean 'list', must've looked at the curly braces